dotnet-interactive/JupyterFormattingKernelExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.CSharp;
+using Microsoft.DotNet.Interactive.FSharp;
+using Microsoft.DotNet.Interactive.Jupyter.Formatting;
+
+namespace Microsoft.DotNet.Interactive.App
+{
+    public static class KernelFormattingExtensions
```

Comment:
I think these should be in `Microsoft.DotNet.Interactive.Jupyter`.

Reply:
...and based on the implementation, `UseLaTeX` would be more accurate and more granular.

Reply:
done